client/components/tinymce/plugins/media/drop-zone.jsx

```diff
@@ -48,10 +48,12 @@ class TinyMCEDropZone extends React.Component {
 
 	componentWillUnmount() {
 		const { editor } = this.props;
-		editor.dom.unbind( editor.getWin(), 'dragenter', this.redirectEditorDragEvent );
 		window.removeEventListener( 'dragleave', this.stopDragging );
-		editor.dom.unbind( editor.getWin(), 'dragleave', this.stopDragging );
-		editor.dom.unbind( editor.getWin(), 'dragover', this.fakeDragEnterIfNecessary );
+		if ( editor.getWin() ) {
```

Comment:
I think we don't even need this; see #20499.

Reply:
Like I stated in the PR description, I only added this check in case there are any use cases where the editor doesn't get destroyed or gets removed after `TinyMCEDropZone`. If you're confident that that won't be the case, then I'm confident to go with your approach of simply deleting the unbind code.